docs/features/software-templates/authorizing-scaffolder-template-details.md

```diff
@@ -225,11 +310,21 @@ class ExamplePermissionPolicy implements PermissionPolicy {
 
 In the provided example permission policy, we only grant the `spiderman` user permissions to perform/access the following actions/resources:
 
-- Read all scaffolder tasks and their associated events/logs.
-- Cancel any ongoing scaffolder tasks.
+- Read scaffolder tasks and their associated events/logs for tasks created by `spiderman`.
+- Read scaffolder task runs of templates owned by `spiderman`
+- Cancel ongoing scaffolder tasks created by `spiderman`.
 - Trigger software templates, which effectively creates new scaffolder tasks.
 
-Any other user would be denied access to these actions/resources.
+On the other hand, the `admin1` user is granted:
```

Comment:
`spiderman` and `admin1`.. how about [venom](https://en.wikipedia.org/wiki/Venom_(character))! Kidding, maybe we make these just Alice and Bob or John and Jane to keep it generic? Unless Bowie makes some more friends!

Reply:
I think it looks great! Just want to make sure @benjdlambert  is also on board (I think bowie needs a sidekick)